### ISSUE TITLE
"Oric-Exos"-like interconnect for multi-PCB hijinx...

### ISSUE THREAD

**Issue by seclorum** [JennyDigital/OriClone-1#1]
Super, super nice! 

Is there any way we can add an interface that would allow an "Oric-Exos" kind of configuration, where one might have a top-layer OricClone-1 with the keyboard, but .. say just for the heck of it .. a stack of 3 PCB's underneath for futher hijinx?

Merging video (see Exos[1]) and a VIA bridge for networking, would be the two specs to think of (power too I suppose) ..

[1] https://forum.defence-force.org/viewtopic.php?f=21&t=1880

I mean, I'm the sort of Oric'ian that will literally order multiple PCB's, once you're ready, if its possible .. :)

**Comment by seclorum** [comment]
See also Twilighte[1] and 

[1] - https://orix.oric.org

**Comment by JennyDigital** [comment]
I'll take a look and see what's involved, and if practical, could add headers and jumpers or whatever.  Do bear in mind that hacks like this tend to make for either towers of PCBs or much larger ones.

On the bright side, there's extra reason for me to re-engineer the PCB anyway as the AY-3-8912 is less than easy to obtain, but the AY-3-8910 is still easy and cheap.

**Comment by seclorum** [comment]
Agreed on all fronts!  The 'tower of PCB's is kind of a unique idea, if you haven't heard of the Oric Exos project.. would allow for some very interesting colour capabilities, and as well provide plenty of processing power for other tasks, if the boards can communicate properly .. anyway just food for thought, I await continued news of your progress with anticipation ..
